### PR TITLE
Top-alignment of blog posts in list #21

### LINKED ISSUE #21: Top-alignment of blog posts in list
At least in my browser, there's something wrong with how the list of blog posts is rendering.

Screen Shot 2020-02-16 at 13 59 12

It looks like the content from post 2 is intruding over the top of post 1 somehow. This may be because I don't always have a header photo in my blog posts?

### PR DESCRIPTION
**Pull request by YorgunKelebek**
Addresses Issue #21  Top-alignment of blog posts in list

Fixes #21

### CODE REVIEW

File: css/app.css [hunk 1]
```diff
@@ -47,10 +47,12 @@ aside h1 {
     width: 90%;
     background: #fff;
     padding: 25px 25px 0px 25px;
-    margin-top: -5rem;
     display: inline-block;
     z-index: 2;
 }
+div.blog-image-container + div.summary-blog-section {
+    margin-top: -5rem !important;
```
Comment (goofballLogic) [inline css/app.css]: Is the !important necessary? Usually a code smell

File: index.html [hunk 2]
```diff
@@ -54,6 +51,12 @@ <h1 class="summary-title"></h1>
             </article>
         </template>
 
+        <template id="blog_image_template">
+            <div class="blog-image-container">
```
Comment (goofballLogic) [inline index.html]: Is this container div strictly necessary? Seems like the IMG tag would be sufficient on its own?

File: js/app.js [hunk 3]
```diff
@@ -74,13 +75,16 @@ function buildBlogSummary(blog, modularContent)
...
-    blogSummary.querySelector(".summary-blog-post").dataset.item_id =
+    const blogPost = blogSummary.querySelector(".summary-blog-post");
+    blogPost.dataset.item_id =
         blogSummary.querySelector(".blog-preview-toggle").dataset.item_id = blog.system.id;
 
     const blogMediaImageURL = firstOrDefaultValue(blog, "blog_media___image", "url");
 	if (blogMediaImageURL)
 	{
-		blogSummary.querySelector(".blog-image").src = blogMediaImageURL;
+        const blogImage = document.importNode(blogImageTemplate.content, true);
```
Comment (goofballLogic) [inline js/app.js]: Something strange going on with whitespace (indentation) here. If it's a problem with tabs Vs spaces we should standardise (probably on spaces)

**Comment by goofballLogic** [comment]
By the way if you use the magic "fixes" or "closes" comment in your PR it will close out an issue for you when you merge it. https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue

File: css/app.css (hunk 1 above)
Comment (YorgunKelebek) [reply css/app.css]: It was originally as I had the selector as `div.blog-image-container + div`

File: js/app.js (hunk 3 above)
Comment (YorgunKelebek) [reply js/app.js]: Looks fine in the code, this is the removed line, but not sure why it is showing like that...

File: index.html [hunk 4]
```diff
@@ -54,6 +51,10 @@ <h1 class="summary-title"></h1>
             </article>
         </template>
 
+        <template id="blog_image_template">
+            <img class="blog-image " />
```
Comment (goofballLogic) [inline index.html]: there's a trailing whitespace at the end of the class name

**Pull request merged**